index.rst

```diff
@@ -80,6 +80,8 @@ DatasetType
 
 The conceptual type of which :ref:`Datasets <Dataset>` are instances (e.g. ``calexp``).
 
+[KSK] I worry about this.  It still is conflated in my mind with ``datatype``, which is a different thing.  In fact there can be many different ``DatasetType``\ s that have the exact same datatype.  I'm not sure what information this particular thing gives you other than a handle to reference the ``Dataset`` with.
```

Comment:
I'm not wedded to any particular names (anywhere in this document), and I'm happy to consider alternatives.

I think maybe the difference of opinion on whether this is a good name comes down to which of the following we think is more important:
 - I think the `Dataset`-`DatasetType`-`DatasetMetaType` relationships are very much *analogous* to the Python `instance`-`class`-`metaclass` relationships (i.e. the internal relationships between the concepts in this document are consistent).
 - But the mapping between the `Dataset` concepts here and the Python instances/types they correspond to is not at all direct (i.e. the external relationships are confusing).

Reply:
I agree that the relationships are important.  My only quibble is that `DatasetType` is really an arbitrary string that we use to link a `Dataset` to it's `DatasetMetaType`.  If you are thinking about something else I'm wrong, but if not I think `DatasetLabel` or `DatasetName` is more clear.

Reply:
You've got a very good point that a `DatasetType` is, in many contexts, just a string, because we usually just need a string as the key in a conceptual dictionary that has the `DatasetMetatype` and the tuple-of-relevant-DataUnit-types as values.  I had been thinking of the `DatasetType` as *including* the latter two things, which is why I had been thinking of it as more than its name, but redefining the concepts around that mapping idea may indeed be less confusing.